sfr/single_assembly/core_support_plate_2d.i

```diff
@@ -28,6 +28,13 @@ Tsf    = 623.15 # stress-free temperature for the ComputeMeanThermalExpansionFun
     top_right = '1e-3 1e-3 0'
     bottom_left = '-1e-3 -1e-3 0'
   []
+  [add_outer_nodeset]
+    type = BoundingBoxNodeSetGenerator
+    input = add_nodeset
+    new_boundary = outer_node
+    top_right = '0.064 1e-3 0'
+    bottom_left = '0.058 -1e-3 0'
```

Comment:
so are we not constraining radial expansion by doing that though? and creating asymmetty?

Reply:
If you pick the node in the same plane and along one of the coordinate axes as 0,0,0 and you fix it in the appropriate direction it should not create asymmetry. If you have perfect symmetry picking a node on the x-axis and fixing it against movement in the y direction doesn't constrain radial expansion. The node is free to move in x, which on the x-axis is the radial direction. For a circle, with symmetric conditions all the nodes on the x and y axis could be fixed (x-axis against moving in y, y-axis against moving in x) as they represent the symmetry plane. You'd get the same results if you modeled a quarter of the circle.  The problem as it stands now is free to rotate unconstrained about the origin.

Reply:
If you are in today, I could give you a quick explanation on a white board about this.

Reply:
I ll be there tomorrow

Reply:
but yes I understand thanks for the explanation